examples/src/main/java/io/dapr/examples/actors/http/DemoActorClient.java

```diff
@@ -29,24 +29,24 @@
    * @throws InterruptedException If program has been interrupted.
    */
   public static void main(String[] args) throws InterruptedException {
-    ActorProxyBuilder<DemoActor> builder = new ActorProxyBuilder(DemoActor.class);
+    try (ActorProxyBuilder<DemoActor> builder = new ActorProxyBuilder(DemoActor.class)) {

```

Comment:
This is a breaking change. I wanted to avoid this, but this is the best place I found to keep the ManagedChannel to be closed for GRPC calls.